dev/io.openliberty.jaxws.config_fat/fat/src/io/openliberty/jaxws/config/fat/WebServiceBndServiceRefPropertiesConfigTest.java

```diff
@@ -0,0 +1,174 @@
+ * to a specific configuration in either the ibm-ws-bnd.xml or server.xml files. Using multiple client instances prevents us from having to reload the server/application by changing
+ * config for the same client instance. 
+ * 
+ * We use the the @see com.ibm.ws.test.stubclient packages from the com.ibm.ws.jaxws.2.2.webcontainer_fat project for this test, with the modifications to 
+ * the @see com.ibm.ws.test.stubclient.client.SimpleStubClientServlet change to use cdi injected @WebServiceRef client, which this configuration requires. 
+ * 
+ * The test uses the @see com.ibm.ws.test.stubclient.SimpleEcho as the Web Service, which simply returns whatever string it was passed in the request back to the client
+ */
+@RunWith(FATRunner.class)
+public class WebServiceBndServiceRefPropertiesConfigTest {
+    @Server("WebServiceRefBndConfigTestServer")
+    public static LibertyServer server;
+
+    private static final String APP_NAME = "simpleTestService";
+
+    private static final String SERVLET_PATH = "/" + APP_NAME + "/SimpleStubClientServlet";
+
+    @BeforeClass
+    public static void setUp() throws Exception {
+
+        ShrinkHelper.defaultApp(server, APP_NAME, "io.openliberty.jaxws.fat.stubclient",
+                                      "io.openliberty.jaxws.fat.stubclient.client");
+        server.startServer();
+        
+
+        assertNotNull("Application hello does not appear to have started.", server.waitForStringInLog("CWWKZ0001I:.*" + APP_NAME));
+    }
+    @AfterClass
+    public static void tearDown() throws Exception {
+        if (server != null && server.isStarted()) {
+            server.stopServer();
+        }
+    }
+    
+    /**
+     * This tests configuring the enableLoggingInOutInterceptor=true in the server.xml file. This tests invokes the test servlet, which uses a @WebServiceRef service client
+     * to send a simple soap request to the SimpleEcho Web Service. 
+     * 
+     * If the configuration is properly applied, the inbound/outbound SOAP Request/Responses will show up in the logs with 
+     * the "REQ_OUT" "RESP_IN". The test verifies expected messages exist in the logs. 
+     * 
+     * 
+     * The test passes serviceRef="server" to tell the SimpleStubClientServlet to use the @WebServiceRef client that's bound to the <service-ref> in the server.xml file
+     * 
+     * @throws Exception
+     */
+    @Test
+    public void testEnableLoggingInOutInterceptorServer() throws Exception {
+
+
+    	   String serviceRef = "server";
+            String response = invokeService(serviceRef); // Call the SimpleStubClientServlet invoke the SimpleService Web Service
+            assertTrue("Expected response to contain Pass but contained: " + response, response.contains("Pass"));
+            
+            // Verify message.log contains the SOAP Message logging
+            List<String> dumpInMessages = server.findStringsInLogs("REQ_OUT");
```

Comment:
Should this be other way around? InMesages (REQ_OUT) and OutMessages (REQ_IN)?

Reply:
This is how the original test was written, I was just keeping the order the same.